doc/source/cluster/kubernetes/examples/rayserve-llm-example.md

```diff
@@ -0,0 +1,140 @@
+(kuberay-rayservice-llm-example)=
+
+# Serve a Large Language Model using Ray Serve LLM on Kubernetes
+
+This guide provides a step-by-step walkthrough for deploying a Large Language Model (LLM) using Ray Serve LLM on Kubernetes. Leveraging KubeRay, Ray Serve, and vLLM, this guide deploys the  `Qwen/Qwen2.5-7B-Instruct` model from HuggingFace, enabling scalable, efficient, and OpenAI-compatible LLM serving within a Kubernetes environment.
+
+## Prerequisites
+
+In this example, model weights are downloaded from Hugging Face. In order to properly finish this guide, you must fulfill the following requirements:
+* A [Hugging Face Account](https://huggingface.co/) and a Hugging Face [Access Token](https://huggingface.co/settings/tokens) with read access to gated repos.
+* In your Ray Serve configuration, set the `HUGGING_FACE_HUB_TOKEN` environment variable to this token to enable model downloads.
+* Any supported GPU. You can refer [here](https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/k8s-cluster-setup.html#kuberay-k8s-setup) for more information
```

Comment:
Move this to a independent step "Step 1: Create a Kubernetes cluster with GPUs".